CellProfiler64.iss

```diff
@@ -101,6 +99,7 @@ End;
 function InitializeSetup(): Boolean;
 Var
   Message: String;
+  JavaVer: String;
```

Comment:
Nope - we check to make sure Java is installed and if not, we tell the user to go get it.